src/lib/widgets/now-playing/ui/episode-display.svelte

```diff
@@ -4,7 +4,10 @@
   export let podcast: Podcast;
 </script>
 
-<img class="flex rounded-2xl" src={podcast.coverUrl} alt="" />
+<div
```

Comment:
question: why is this necessary?

Reply:
There was a problem of image always taking the full width of the container and scaling vertically to enormous sizes. I wanted to try solving it with `img` tag but failed, so the `background` came up to place perfectly.